addons/traefik/1.7.x/tests/metallb-config.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+JP='{.status.addresses[?(.type == "InternalIP")].address}'
+NODEIP=$(kubectl get node kind-control-plane -o jsonpath="${JP}")
+SLASH24=$(sed 's@\.[0-9]\+$@@' <<< ${NODEIP})
+kubectl -n kubeaddons create configmap metallb-config --from-literal=config='{"address-pools":[{"name":"default","protocol":"layer2","addresses":["'${SLASH24}'.200-'${SLASH24}'.250"]}]}'
+
```

Comment:
wdyt about making this script pass `shellcheck`?

current output:

```
shellcheck metallb-config.sh 

In metallb-config.sh  line 5:
SLASH24=$(sed 's@\.[0-9]\+$@@' <<< ${NODEIP})
          ^-- SC2001: See if you can use ${variable//search/replace} instead.
                                   ^-------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
SLASH24=$(sed 's@\.[0-9]\+$@@' <<< "${NODEIP}")


In metallb-config.sh  line 6:
kubectl -n kubeaddons create configmap metallb-config --from-literal=config='{"address-pools":[{"name":"default","protocol":"layer2","addresses":["'${SLASH24}'.200-'${SLASH24}'.250"]}]}'
                                                                                                                                                    ^--------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                                                                                                                     ^--------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
kubectl -n kubeaddons create configmap metallb-config --from-literal=config='{"address-pools":[{"name":"default","protocol":"layer2","addresses":["'"${SLASH24}"'.200-'"${SLASH24}"'.250"]}]}'

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
  https://www.shellcheck.net/wiki/SC2001 -- See if you can use ${variable//se...
```